conbench/entities/distribution.py

```diff
@@ -172,14 +172,14 @@ def set_z_scores(benchmark_results: List["BenchmarkResult"]):
     """
     # Cache Distribution query results by commit/hardware, in case multiple input
     # benchmark_results (e.g. from the same Run) share those attributes
-    cached_distributions = {}
+    cached_distributions = {}  # type: ignore[var-annotated]
```

Comment:
Types might be useful here. :) This one should be `Dict[Tuple[str, str], Dict[Tuple[str, str], Tuple[Optional[float], Optional[float]]]]`

(haha maybe that's too complex to be useful?)

Reply:
Yeah I thought the same that adding type annotations to that function will be useful, for sure! :) Thanks for thinking through those already. Let's do those in separate patches.